.github/workflows/cran-status-check.yml

```
@@ -0,0 +1,22 @@
+name: check-cran-status
+
+on:
+  schedule:
+    - cron: '0 6 * * 1,3,5' # Runs at 06:00 on Mon/Wed/Fri
+    
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    permissions: write-all # allows it to create issues
```

```suggestion
    permissions:
      issues: write
```

I believe this should be sufficient(?)